test/system/Code/content_test.rb

```diff
@@ -0,0 +1,200 @@
+require "application_system_test_case"
+
+class ContentTest < ApplicationSystemTestCase
+  setup do
+    Capybara.app_host = "https://alpha.vulnerabilityhistory.org"
+    Capybara.run_server = false
+
+    @start = filepaths_path
```

Comment:
I think for simplicity's sake we can just do this:

```suggestion
    @start = '/filepaths'
```